packages/integration-tests/tests/api/contracts.test.ts

```diff
@@ -22,6 +22,61 @@ describe("API module: Contract", () => {
       await playbook.deployMultiCallContracts();
     });
 
+    //id1851
+    it("Verify /api?module=contract&action=checkverifystatus response", async () => {
+      await helper.retryTestAction(async () => {
+        apiRoute = `/api?module=contract&action=checkverifystatus&guid=3177`;
+        response = await helper.performGETrequest(apiRoute, "sepolia");
+        console.log(response.body);
+
+        expect(response.status).toBe(200);
+        expect(response.body).toStrictEqual(expect.objectContaining({ status: "1" }));
+        expect(response.body).toStrictEqual(expect.objectContaining({ message: "OK" }));
+        expect(response.body).toStrictEqual(expect.objectContaining({ result: "Pass - Verified" }));
+      });
+    });
+
+    //id1695
+    it("Verify /api?module=contract&action=getabi response", async () => {
+      await helper.retryTestAction(async () => {
+        const greeterContract = "0x180faDac8ce638C67640a6C29BdfB5CB60eCDB76";
```

Comment:
please move it to entities:

export enum Contracts {
  greeterContract = "0x180faDac8ce638C67640a6C29BdfB5CB60eCDB76",
}